include/seqan3/io/alignment_file/format_sam.hpp

```diff
@@ -488,14 +488,14 @@ class alignment_file_input_format<format_sam>
      */
     template <typename align_type, typename ref_seqs_type>
     void construct_alignment(align_type                           & align,
-                             std::vector<std::pair<char, size_t>> & cigar,
+                             std::vector<cigar>                   & cigar_vector,
```

Comment:
Please adapt doxygen's `\param[in]` statement to the renamed cigar_vector parameter.